<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCSnippetsForm</class>
 <widget class="QDialog" name="GCSnippetsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Snippet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.jpg</normaloff>:/resources/goblinicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Overrides main style sheet. */
QCheckBox
{
  border-radius: 1px;
}

QCheckBox::indicator
{
  subcontrol-position: center;
  padding-left: 7;
  border-radius: 1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="indentation">
       <number>12</number>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>50</number>
      </attribute>
      <column>
       <property name="text">
        <string>Elements:</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>27</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Attributes:</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Default Values:</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Incr:</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showHelpButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/gcresources.qrc">
         <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
